websockets/server.py

```diff
@@ -232,7 +232,9 @@ def get_response_status(self, set_header):
 
         This method may be overridden to check the request headers and set a
         different status, for example to authenticate the request and return
-        ``HTTPStatus.UNAUTHORIZED`` or ``HTTPStatus.FORBIDDEN``.
+        ``HTTPStatus.UNAUTHORIZED`` or ``HTTPStatus.FORBIDDEN``. The
+        current request path can be accessed from within this method
+        as ``self.path``.
```

Comment:
A section like that in the docs makes sense, though I think I'd like to wait until after #116 is resolved to do that since it may require rewriting a portion those docs.

I do think the fact that `self.path` and `self.request_headers` are guaranteed to be available should be stated in the same place developers are told the method is overridable (currently stated in the docstring). That way implementers are clearer on what they can / should be relying on when they override.